local/pid/pidfile.go

```diff
@@ -258,15 +259,24 @@ func (p *PidFile) LogReader() (io.ReadCloser, error) {
 }
 
 func (p *PidFile) LogWriter() (io.WriteCloser, error) {
+	// instantiate the log writer only once per process lifetime, this is useful
+	// to have a single truncate (and thus a clean log file) at the beginning of
+	// the process management but not truncate the log file when the process is
+	// restarted.
+	if p.lw != nil {
```

Comment:
Does it need to be guarded by a sync.Once?

Reply:
done but moving to draft because I didn't had the opportunity to run it locally nor run the test suite

Reply:
just tried it locally and worked as expected